doc/gnucobol.texi

```diff
@@ -445,26 +445,26 @@ Micro Focus' Variable format.  Identical to the fixed format above
 except for the program-text area which extends up to column 250 instead
 of 72.
 
-@item -fformat=xopen
-X/Open Free-form format.  The program-text area may start in column 1
-unless an indicator is present, and lines may contain up to 80
-characters.  Indicator for debugging lines is @code{D } instead of
-@code{D} or @code{d}.
-
 @item -fformat=xcard
 ICOBOL xCard format.  Variable format with right margin set at column
 255 instead of 250.
 
+@item -fformat=xopen
+X/Open Free-form format.  The program-text area may start in column 1
+unless an indicator is present, and lines may contain up to 255
+characters.  Indicator for debugging lines is ``@code{D }'' instead of
+``@code{D}'' or ``@code{d}''.
+
 @item -fformat=crt
 ICOBOL Free-form format (CRT).  Similar to the X/Open format above, with
 lines containing up to 320 characters and single-character debugging
-line indicators (@code{D} or @code{d}).
+line indicators (``@code{D}'' or ``@code{d}'').
```

Comment:
I added the quotes as when I looked at what `@code{D }` (D followed by a space) rendered in the PDF I found it was not clear enough. So that was an attempt to clarify. The other bits of code do not use the quotes: an alternative would be to just put them for the case with space followed by the clarification in parenthesis.

Reply:
rechecked in the gdb manual: they use [`@samp{D }`](https://www.nongnu.org/texi2latex/texinfo/samp.html) in this case. Let's do the same.

If you find the time to do so an overall change would obviously be better than only doing it here ;-) but I understand if this comes later.